test/build/unknown/__snapshots__/unknown.test.js.snap.webpack5

```diff
@@ -37,56 +37,55 @@ exports[`unknown behaviour should log an error if an unknown flag is passed and
 
 exports[`unknown behaviour should log an error if an unknown flag is passed and suggests the closest match to an unknown flag #2: stderr 1`] = `
 "[webpack-cli] Error: Unknown option '--output-fileneme'
-[webpack-cli] Did you mean '--output-filename'?
+(Did you mean --output-filename?)
 [webpack-cli] Run 'webpack --help' to see available commands and options"
 `;
 
 exports[`unknown behaviour should log an error if an unknown flag is passed and suggests the closest match to an unknown flag #2: stdout 1`] = `""`;
 
 exports[`unknown behaviour should log an error if an unknown flag is passed and suggests the closest match to an unknown flag #3: stderr 1`] = `
 "[webpack-cli] Error: Unknown option '--output-library-auxiliary-comment-commnjs'
-[webpack-cli] Did you mean '--output-library-auxiliary-comment-commonjs'?
-[webpack-cli] Did you mean '--output-library-auxiliary-comment-commonjs2'?
+(Did you mean --output-library-auxiliary-comment-commonjs?)
 [webpack-cli] Run 'webpack --help' to see available commands and options"
 `;
 
 exports[`unknown behaviour should log an error if an unknown flag is passed and suggests the closest match to an unknown flag #3: stdout 1`] = `""`;
 
 exports[`unknown behaviour should log an error if an unknown flag is passed and suggests the closest match to an unknown flag using "b" command: stderr 1`] = `
 "[webpack-cli] Error: Unknown option '--entyr'
-[webpack-cli] Did you mean '--entry'?
+(Did you mean --entry?)
 [webpack-cli] Run 'webpack --help' to see available commands and options"
 `;
 
 exports[`unknown behaviour should log an error if an unknown flag is passed and suggests the closest match to an unknown flag using "b" command: stdout 1`] = `""`;
 
 exports[`unknown behaviour should log an error if an unknown flag is passed and suggests the closest match to an unknown flag using "bundle" command: stderr 1`] = `
 "[webpack-cli] Error: Unknown option '--entyr'
-[webpack-cli] Did you mean '--entry'?
+(Did you mean --entry?)
```

Comment:
Will be great to have `[webpack-cli]` prefix here, is there the places where we can hook?

Reply:
Yes, we can disable it with `this.program.showSuggestionAfterError(false);` and keep our own logic.